src/dates.js

```diff
@@ -1,3 +1,4 @@
 export { default as formatDate } from './dates/format-date';
 export { default as parseDate } from './dates/parse-date';
-export { default as dateFormatString } from './dates/date-format-string';
\ No newline at end of file
+export { default as dateFormatString } from './dates/date-format-string';
```

Comment:
Don't forget to re-export these functions from `kendo-angular-intl` because of CDN bundle. :)